docusaurus/docs/Android/04-compose/04-message-components/02-message-list-header.mdx

```diff
@@ -93,17 +96,27 @@ As the component is fully state-dependent, you can customize the data it shows,
 fun MessageListHeader(
     channel: Channel,
     currentUser: User?,
-    connectionState = ConnectionState.CONNECTED,
-    messageMode: MessageMode = Normal,
-    ..., // content
-    ... // actions
+    modifier: Modifier = Modifier,
+    typingUsers: List<User> = emptyList(),
+    messageMode: MessageMode = MessageMode.Normal,
+    connectionState: ConnectionState = ConnectionState.CONNECTED,
+    color: Color = ChatTheme.colors.barsBackground,
+    shape: Shape = ChatTheme.shapes.header,
+    elevation: Dp = ChatTheme.dimens.headerElevation,
+    ..., // Handlers
+    ... // Content
 )
 ```
 
 * `channel`: The information about the current channel, used to show the member count, name and avatar data.
 * `currentUser`: Currently logged in user, used to differentiate it from other users, when loading the channel image.
-* `connectionState`: The state of WebSocket connection, used to switch between the member count text and the the `NetworkLoadingView`.
+* `modifier`: Modifier for the root component. You can use it to add padding or change the dimensions and such, however attributes such as color, shape and elevation have their own parameters.
+* `typingUsers`: List of typing users used to show their names along with a typing indicator.
 * `messageMode`: Used to determine the header title. If we're in a thread, we show a title saying who the owner of the parent message is.
+* `connectionState`: The state of WebSocket connection, used to switch between the member count text and the network status information.
```

Comment:
I would remove the `WebSocket` word here - as users don't need to know that

Reply:
Yes, I think removing 'WebSocket' is a good choice.

For the "watcher" vs "member" dilemma please see [this](https://github.com/GetStream/stream-chat-android/pull/2855#discussion_r777556287) thread.

Reply:
If we remove the socket could we lose the whole first part of the sentence?

I feel like "`connectionState`:  The state of the connection" might be a little bit redundant.

Reply:
`connectionState: Used to switch between the member count text and the network status information.`

is good